src/service/alert.service.ts

```diff
@@ -29,7 +29,7 @@ export class AlertService {
     }
 
     clear() {
-        this.alerts = [];
+        this.alerts.forEach((alert: Alert) => setTimeout(() => alert.close(), 0));
```

Comment:
Im wondering why clearing the array doesnt work. Components are suppose to be model driven. May be we need to find the real issue